apps/site/assets/css/_utilities.scss

```diff
@@ -57,6 +57,12 @@
 .u-bg--bus {
   @include bg-text($black);
   background-color: $brand-bus;
+  .c-link-block__inner-link {
+    border: 1px solid $black;
+    svg {
+      margin-top: 1px;
+    }
+  }
```

Comment:
My main issue is I'd rather keep the `_utilities.scss` file simple, and have this declaration in one of the other CSS files (`_link-block.scss`, `_transit-near-me.scss`, or wherever else).